components/advocates/AdvocateCard.vue

```diff
@@ -1,79 +1,66 @@
 <template>
-  <article class="advocate-card">
-    <div
-      class="advocate-card__picture"
-      :style="`background-image: ${decorate(image)};`"
-    />
-    <div class="advocate-card__copy">
-      <h3 class="advocate-card__name">
-        {{ name }}
-      </h3>
+  <AppCard
+    class="advocate-card"
+    :image="image"
+    :title="name"
+    :tags="formattedRegion"
+  >
+    <div>
```

Comment:
I don't think we need this `div` anymore

Reply:
I will check it tomorrow. Good catch.